packages/application-shell/src/test-utils/custom-views-utils.tsx

```diff
@@ -45,6 +49,10 @@ export const renderCustomView = (
       locale: props.locale,
       project: {
         key: props.projectKey,
+        allAppliedPermissions: props.projectAllAppliedPermissions || [],
```

Comment:
Writing the docs I found out we allow for custom permissions in Custom Application testing so I'm adding that possibility for Custom Views as well.